packages/react-native-gesture-handler/src/v3/hooks/useGestureEvent.ts

```diff
@@ -0,0 +1,36 @@
+import { useGestureStateChangeEvent } from './events/useGestureStateChangeEvent';
+import { useGestureHandlerEvent } from './events/useGestureHandlerEvent';
+import { useTouchEvent } from './events/useTouchEvent';
+
+export function useGestureEvent(
+  handlerTag: number,
+  config: any,
+  shouldUseReanimated: boolean
+) {
+  const onGestureHandlerStateChange = useGestureStateChangeEvent(
+    handlerTag,
+    config,
+    shouldUseReanimated
+  );
+  const onGestureHandlerEvent = useGestureHandlerEvent(
+    handlerTag,
+    config,
+    shouldUseReanimated
+  );
+
+  const onGestureHandlerTouchEvent = useTouchEvent(
+    handlerTag,
+    config,
+    shouldUseReanimated
+  );
+
+  const onGestureHandlerAnimatedEvent =
+    config.onGestureHandlerAnimatedEvent as (...args: any[]) => void;
```

Comment:
In my head, `onGestureHandlerAnimatedEvent` is set automatically when `onUpdate` is set to an `Animated.Event`. Please add a TODO if you don't have any objections.